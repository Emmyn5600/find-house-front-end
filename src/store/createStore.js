import { createStore, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import reducer from './reducers/index';

const store = () => createStore(reducer, applyMiddleware(thunk));

export default store;
