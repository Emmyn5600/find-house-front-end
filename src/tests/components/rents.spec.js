import { BrowserRouter, Route } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import createStore from '../../store/createStore';

import RentLists from '../../container/RentsList';

const store = createStore();

describe('RentsList', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Provider store={store}>
            <Route
              path="/"
              component={(props) => <RentLists history={props.history} />}
            />
          </Provider>
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
