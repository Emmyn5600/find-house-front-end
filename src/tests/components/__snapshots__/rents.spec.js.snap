// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RentsList should render correctly 1`] = `"List is empty"`;
